guides/common/assembly_hammer-reference.adoc

```diff
@@ -1,3 +1,5 @@
+:_mod-docs-content-type: ASSEMBLY
+
 // DO NOT EDIT MANUALLY
```

Comment:
I think this will be overwritten by `generate-hammer-reference.sh`. OK for now, but maybe we need a check or adjustment to this shellscript to not drop this change when we recreate the Hammer CLI reference.

Reply:
Good point. I forgot about auto-generated files. Will fix.

Reply:
Thanks for catching this! I would prefer to update the script as part of this PR.